src/main.rs

```diff
@@ -42,14 +40,25 @@ fn main() {
     // Generate subscription id, if not provided
     let subscription_id = match or_list.first() {
         Some(args) => {
-          let cli_matches = cli().get_matches_from(args);
-          match cli_matches.get_one::<String>("subscription-id") {
-            None => { Uuid::new_v4().to_string() },
-            Some(m) => m.to_string()
-          }
-        },
-        None => { Uuid::new_v4().to_string() }
+            let cli_matches = cli().get_matches_from(args);
+            match cli_matches.get_one::<String>("subscription-id") {
+                None => Uuid::new_v4().to_string(),
+                Some(m) => m.to_string(),
+            }
+        }
+        None => Uuid::new_v4().to_string(),
+    };
+
+    let method = if cli().get_matches_from(args).get_flag("count") {
```

Comment:
I prefer to use match statements instead of if/else, and if we expand commands in future (for example AUTH, POW), it would be nice to already be a match statement.

let method = match cli().get_matches_from(args).get_flag("count") {
    true => "COUNT",
    false => "REQ",
};